---
#是否检测到停机坪
bool flag
#停机坪中心像素坐标
int16 x
int16 y